blakserv/ccode.c

```diff
@@ -2344,3 +2345,45 @@ blak_int C_MinigameStringToNumber(int object_id,local_var_type *local_vars,
 	
 	return ret_val.int_val;
 }
+
+blak_int C_GetServerConfigValue(int object_id,local_var_type *local_vars,
+				int num_normal_parms,parm_node normal_parm_array[],
+				int num_name_parms,parm_node name_parm_array[])
+{
+	val_type config_id, ret_val;
+		
+	config_id = RetrieveValue(object_id,local_vars,normal_parm_array[0].type,
+		normal_parm_array[0].value);
+  
+	if (config_id.v.tag != TAG_INT)
+	{
+		bprintf("C_GetServerConfigValue can't set from non-int %i,%i\n",
+			config_id.v.tag,config_id.v.data);
+		return NIL;
+	}
+
+	config_node *cnode = GetConfigByID(config_id.v.data);

```

Comment:
This seems way too permissive, as it allows Blakod to read any config value that exists now or in the future.  Let's lock it down to the one(s) we actually want to allow.